.github/workflows/kind-e2e.yaml

```diff
@@ -53,7 +53,8 @@ jobs:
       DAPR_NAMESPACE: dapr-tests
       GOVER: "^1.19.2"
       # Container registry where to cache e2e test images
-      DAPR_CACHE_REGISTRY: "dapre2eacr.azurecr.io"
+      # Disabling it for release-1.1- branch.
+      DAPR_CACHE_REGISTRY: "localhost:5000/dapr"
```

Comment:
This shouldn't be necessary - images are cached and referenced  by their sha256 so

Reply:
Yes, this was me trying an hypothesis of the caching being buggy.